apps/antalmanac/src/lib/download.ts

```diff
@@ -246,74 +248,84 @@ export function getRRule(bydays: string[], quarter: string) {
     return `FREQ=WEEKLY;BYDAY=${bydays.toString()};INTERVAL=1;COUNT=${count}`;
 }
 
-export function getEventsFromCourses(courses = AppStore.schedule.getCurrentCourses()): EventAttributes[] {
-    const events = courses.flatMap((course) => {
-        const {
-            term,
-            deptCode,
-            courseNumber,
-            courseTitle,
-            section: { sectionType, instructors, meetings, finalExam },
-        } = course;
-
-        const courseEvents: EventAttributes[] = meetings
-            .map((meeting) => {
-                if (meeting.timeIsTBA) {
-                    return;
+export function getEventsFromCourses(events = AppStore.getEventsInCalendar()): EventAttributes[] {
+    const calendarEvents = events.flatMap((event) => {
+        if (!event.isCustomEvent) {
```

Comment:
Could you swap the two blocks of conditional logic? It can be easier to read "positive" conditions.

For instance, given the two conditions: "if today is not raining" vs. "if today is sunny", I'd prefer the latter. It depends on the scenario, but I think it makes sense here. So the pseudocode would be "if it's a custom event, process the item as a custom event, otherwise process it as a course event".

```suggestion
        if (event.isCustomEvent) {
```